docs/matchers/regex.md

```diff
@@ -91,3 +91,22 @@ async def remember(opsdroid, config, message):
     await opsdroid.memory.put("remember", remember)
     await message.respond("OK I'll remember that")
 ```
+
+## Score factor
+
+In order to make NLU skills execute over regex skills, opsdroid always applies a factor of `0.6` to every regex evaluated score.
+
+Sometimes can be interesting for a developer to get a regex skill executed over a NLU skill, and `score_factor` keyword argument can be used to archieve this.
```

Comment:
I would restructure this sentence like this:
"If a developer want to have a regex skill executed over a NLU one then the keyword argument `score_factor`  can be used to achieve this."